rps.js

```diff
@@ -0,0 +1,41 @@
+const userChoice = prompt("Do you choose rock, paper or scissors?");
+
+var computer= Math.random();
+
+ if (computer < 0.30) {
+    computer = 'rock';
+ } else if(computer <= 0.42) {
+    computer = 'paper';
+ } else {
+    computer = 'scissors';
+ } 
+
+const playGame = function(userChoice,computer){
+let rseult;
+
+    if (userChoice === computer){
+       return "it is a tie";
+    }
+if(userChoice ==="rock" && computer === "scissar")
+{
+    return "winner";
+
+} else{
+    return "You lose! Try again.";
+}
```

Comment:
This will return you lose, before checking out the other conditions, even if the user was winning.
Try it with making the user choice `paper` and see what you get.
Think about how you can make your program check all conditions.